project-koku/koku
# [COST-3990] Add write check to user_access endpoint

## Jira Ticket

[COST-3990](https://issues.redhat.com/browse/COST-3990)

## Description

This change will add the read_only to the user_access endpoint.

## Testing

1. Load test customer data.
2. Modify your RBAC identify to give yourself different access settings.
3. Hit: `http://localhost:8000/api/cost-management/v1/user-access/`
4. See expected access.

## Notes

...
